tasks/main.yml

```diff
@@ -8,7 +8,7 @@
 - include_vars: "vars/{{ ansible_os_family }}.yml"
 
 - name: Run {{ letsencrypt_webserver }}.yml playbook
-  include: "{{ letsencrypt_webserver }}.yml"
+  import_tasks: "{{ letsencrypt_webserver }}.yml"
```

Comment:
The difference between include and import isn't trivial. It's fine to update the syntax to `include_tasks`, but unless you've tested this change, I reckon switching to import will break things.